pine-a64-plus/overlays/armbian/boot/armbianEnv.txt

```diff
@@ -0,0 +1,7 @@
+verbosity=1
+bootlogo=false
+console=both
+disp_mode=1920x1080p60
+overlay_prefix=sun50i-a64
+rootdev=UUID=0129b70b-dfe6-4c6c-9fa1-8683a2f2dfdb
```

Comment:
I think it’d be better to use a label to find the root device; alternatively, generate the file during the build with a script and insert the actual UUID here.

Reply:
Yes, its a really weak point of my build :)